middleware.ts

```
@@ -1,11 +1,18 @@
 import { clerkMiddleware, createRouteMatcher } from '@clerk/nextjs/server';
 
+// Define protected routes
 const isProtectedRoute = createRouteMatcher([
   '/dashboard(.*)',
   '/admin(.*)',
   '/api/protected(.*)'
 ]);
 
+// Define auth routes
+const isAuthRoute = createRouteMatcher([
```

ESLint

[error] 'isAuthRoute' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)